### ISSUE TITLE
Intents authorize will behave the same as a purchase by default

### ISSUE THREAD

**Issue by jamieburchell** [thephpleague/omnipay-stripe#232]
Removing `$data['capture_method'] = 'manual'` [here](https://github.com/thephpleague/omnipay-stripe/commit/033ea3e6f3d1ea75c50a78b8d88fcab84afa27b7#r139548793) changes this method to behave the same as a purchase by default and not an authorization. The authorization method is supposed to require a manual capture as per the class comments:

```
 * An authorize request is similar to a purchase request but the
 * charge issues an authorization (or pre-authorization), and no money
 * is transferred.  The transaction will need to be captured later
 * in order to effect payment. Uncaptured charges expire in 7 days.
```

**Comment by jamieburchell** [comment]
Attn: @barryvdh I believe this is a breaking change that by default will change the nature of the `authorize` method to become a regular `payment` method instead.

**Comment by barryvdh** [comment]
Yeah so do you have a PR to fix it?

**Comment by jamieburchell** [comment]
> Yeah so do you have a PR to fix it?

No. I just spotted this when trying to make sense of the readme vs code comments vs actual code and thought it was a serious issue.

I don't think it makes sense to allow the `capture_method` to be set at all. Authorization request == manual, payment request == automatic?
